module/Decision/src/Decision/Controller/FileBrowser/DummyReader.php

```diff
@@ -0,0 +1,65 @@
+<?php
```

Comment:
Is this file still required?

Reply:
Not necessarily. Could be useful for local UI testing in case the given dir is empty.

Reply:
For testing, perhaps set the default path in the config to some harmless path, e.g. getcwd() . /public/